src/main/java/io/neonbee/config/NeonBeeConfig.java

```diff
@@ -34,6 +34,11 @@
      */
     public static final String DEFAULT_TRACKING_DATA_HANDLING_STRATEGY = TrackingDataLoggingStrategy.class.getName();
```

Comment:
unrelated: 
methodes are not used: setEventBusTimeout, setEventBusCodecs, setTrackingDataHandlingStrategy, setPlatformClasses, setTimeZone
public methods not tested: setTimeZone, getTimeZone, setPlatformClasses, setTrackingDataHandlingStrategy, setEventBusCodecs, getEventBusCodecs, setEventBusTimeout, getEventBusTimeout